src/components/places/place.tsx

```diff
@@ -0,0 +1,137 @@
+import { COLORS } from "@/constants/css";
+import styled from "@emotion/styled";
+import Image from "next/image";
+import BookmarkBorderIcon from "@mui/icons-material/BookmarkBorder";
+import IosShareIcon from "@mui/icons-material/IosShare";
+
+interface PlaceProps {
+  businessName: string;
+  address: string;
+}
+
+const SHARE_TEXT = "공유";
+const BOOKMARK_TEXT = "저장";
+
+const Place = ({ businessName, address }: PlaceProps) => {
+  const tmpImageList = [1, 2, 3, 4];
+  return (
+    <Container>
+      <TitleIconContainer>
+        <TitleContainer>
+          <h3>{businessName}</h3>
+          <p>{address}</p>
+        </TitleContainer>
+        <IconsContainer>
+          <div>
+            <IosShareIcon
+              sx={{
+                display: "block",
+                height: "2.1rem",
+                width: "2.1rem",
+              }}
+            />
+            <p>{SHARE_TEXT}</p>
+          </div>
+          <div>
+            <BookmarkBorderIcon
+              sx={{
+                display: "block",
+                height: "2.3rem",
+                width: "2.3rem",
+              }}
+            />
+            <p>{BOOKMARK_TEXT}</p>
+          </div>
+        </IconsContainer>
+      </TitleIconContainer>
+      <ImageContainer>
+        {tmpImageList.map((i, index) => (
+          <Image
+            key={index}
+            src={`/cafe${(i % 3) + 1}.png`}
+            alt='cafe image'
+            width={125}
+            height={125}
+            priority
+          />
+        ))}
+      </ImageContainer>
```

Comment:
이미지 border-radius를 주기 위해 `div`로 감싸기 
```suggestion
        {tmpImageList.map((i, index) => (
          <div key={index}>
          <Image
            src={`/cafe${(i % 3) + 1}.png`}
            alt='cafe image'
            width={125}
            height={125}
            priority
          />
          </div>
        ))}
      </ImageContainer>
```